shell/src/main/java/alluxio/cli/fs/FileSystemShellUtils.java

```diff
@@ -218,6 +220,23 @@ public static String validatePath(String path, AlluxioConfiguration alluxioConf)
     return res;
   }
 
+  /**
+   * Gets the value of an option from the command line.
+   *
+   * @param cl command line object
+   * @param option the option to check for in the command line
+   * @param defaultValue default value for the option
+   * @return argument from command line or default if not present
+   */
+  public static int getIntArg(CommandLine cl, Option option, int defaultValue) {
+    int arg = defaultValue;
+    if (cl.hasOption(option.getLongOpt())) {
```

Comment:
does this work with both short and long options?

Reply:
Yes. This is the same logic we used previously in the persist command for parallelism. I just generalized it